corehq/apps/user_importer/importer.py

```diff
@@ -401,7 +419,7 @@ def create_or_update_users_and_groups(domain, user_specs, upload_user, group_mem
                         user.reset_locations(location_ids, commit=False)
 
                 if role:
-                    role_qualified_id = roles_by_name[role].get_qualified_id()
+                    role_qualified_id = domain_info.roles_by_name[role].get_qualified_id()
```

Comment:
Noticed while reading through this whole function: I think the usage of f strings in the web user errors below won't work with translations. Can you update them?

https://stackoverflow.com/questions/49797658/how-to-use-gettext-with-python-3-6-f-strings

Reply:
fixed